advanced/Scripts/update.sh

```diff
@@ -62,7 +66,7 @@ GitCheckUpdateAvail() {
   fi
 
   # Change back to original directory
-  cd "${curdir}"
+  cd "${curdir}" || exit
```

Comment:
That whole function could use a rewrite, there's no need to store the PWD as the function happens in a subshell so we don't need to move back to the PWD, it's not needed. But that can come in another PR.